src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+
+public class DeadlineTest {
+    @Test
+    public void encodeTaskForStorage_notDoneDeadlineTask_encodedString() {
+        assertEquals("D | 0 | testingDeadlineTaskOne | 2021-08-26",
+                new Deadline("testingDeadlineTaskOne",
+                        LocalDate.of(2021, 8, 26),
+                false)
```

Comment:
Indentation of false here should be consistent with the rest of the test cases. +4 more spaces to the right